docs/configuration/queries.md

```diff
@@ -94,40 +95,50 @@ tasks:
       # ... additional worker properties
 ```
 
-## Pattern Queries
-The pattern attribute has the following properties:
-- `endpoint` - the endpoint to query
-- `limit` - the maximum number of instances per query pattern
-- `save` - if set to `true`, query instances will be saved in a separate file
+## Query Templates
+Query templates are queries containing placeholders for some terms. 
+Replacement candidates are identified by querying a given endpoint. 
+This is done in a way that the resulting queries will yield results against endpoints with the same data.
 
-Example of query configuration with pattern queries:
-```yaml
-queries:
-  path: "./example/suite/queries/"
-  format: "folder" 
-  pattern:
-    endpoint: "http://dbpedia.org/sparql"
-    limit: 100
-    save: true
-```  
+The placeholders are written in the form of `%%var[0-9]+%%`, where `[0-9]+` represents any number.
```

Comment:
As discussed on Tuesday that should rather be something like `%%[a-zA-Z0-9_]+%%`. 

Also mention, that even if the placeholder is equal to a variable name it will not be assigned the same variable name during candidate generation.